lib/Helper/Wizard.ts

```diff
@@ -6,20 +6,16 @@ import type { BaseIntegration } from '../Steps/Integrations/BaseIntegration';
 import { BottomBar } from './BottomBar';
 import { debug, dim, nl, red } from './Logging';
 
-function sanitizeAndValidateArgs(argv: Args): void {
+function sanitizeAndValidateArgs(argv: Args & Record<string, unknown>): void {
```

Comment:
Widening the type here allows us to get rid of a couple of ts-ignores. Not the prettiest fix but this is the legacy wizard code which I con't care too much about.